packages/react-native/Libraries/Text/TextInput/Multiline/RCTUITextView.h

```diff
@@ -39,6 +39,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong, nullable) NSString *inputAccessoryViewID;
 @property (nonatomic, strong, nullable) NSString *inputAccessoryViewButtonLabel;
 
+@property (nonatomic, assign) BOOL isFirstRender;
+@property (nonatomic, strong) NSArray<UIBarButtonItemGroup *> *initialValueLeadingBarButtonGroups;
+@property (nonatomic, strong) NSArray<UIBarButtonItemGroup *> *initialValueTrailingBarButtonGroups;
```

Comment:
This doesn't need to be leaked to the public API, if you really need to store it you can store them privately: 

```
@implementation ClassName {
    int privateInteger;
    MyObject *privateObject;
}
```

Reply:
Done - aa3fbcabf96853232d4ee359afe7942002e8154d, 374a7b50a06a7f6b89270167b40552539e114a5d.